cibuildwheel/linux.py

```diff
@@ -42,8 +43,13 @@ def matches_platform(identifier):
     return False
 
 
-def get_python_configurations(build_selector):
-    PythonConfiguration = namedtuple('PythonConfiguration', ['version', 'identifier', 'path'])
+class PythonConfiguration(NamedTuple):
+    version: str
+    identifier: str
+    path: str
+
+
+def get_python_configurations(build_selector: Callable[[str], bool]) -> List[PythonConfiguration]:
```

Comment:
I think we'd be better to use BuildSelector as a type here - it'll help readability for people new to the codebase.

Reply:
OK, yeah, that might be a good argument. Doing so.